src/components/course/CourseSkills.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useContext, useState } from 'react';
+import { Badge, Button } from '@edx/paragon';
+
+import { CourseContext } from './CourseContextProvider';
+import { SKILLS_BUTTON_TEXT } from './data/constants';
+
+export default function CourseSkills() {
+  const { state } = useContext(CourseContext);
+  const { skillNames } = state.course;
+  const [showMore, setShowMore] = useState(false);
+  const maxVisibleSkills = 5;
+  const skillsButtonText = showMore ? SKILLS_BUTTON_TEXT.SHOW_LESS : SKILLS_BUTTON_TEXT.SHOW_MORE;
+
+  return (
+    <div className="mb-4">
+      <h6> Skills you&apos;ll gain</h6>
+      <div>
+        {skillNames.map((skill, index) => (
+          <Badge
+            key={skill.id}
+            className="course-skill"
+            variant="light"
+            style={{ display: index < maxVisibleSkills || showMore ? 'inline-block' : 'none' }}
```

Comment:
nit: I think it works fine but I prefer usually to parenthesize the conditions in ternary operator expressions (also mainly to increase readability)